tests/test_reblock.py

```diff
@@ -0,0 +1,486 @@
+import geopandas as gpd
+from shapely.geometry import LineString, Polygon
+from shapely.wkt import loads
+import unittest
+from logging import basicConfig, debug, info
+
+from prclz.reblock import planar_graph
+#from prclz.reblock.planar_graph_utils import update_edge_types
+# import reblock2
+# from path_cost import FlexCost
+'''
+To-Do:
+ - Do another round of refactoring
+ - Get the simplify linestring part working, and tested
+ - Build out the momepy parcelization pipeline, and connect
+   that to our reblocking 
+'''
+
+
+def make_square(lower_left_pt, w=1):
+    pts = []
+    x,y = lower_left_pt
+    i,j = 0,0   
+    pts.append((x + w*i, y + w*j))
+
+    i,j = 1,0   
+    pts.append((x + w*i, y + w*j))
+
+    i,j = 1,1   
+    pts.append((x + w*i, y + w*j))
+
+    i,j = 0,1   
+    pts.append((x + w*i, y + w*j))
+    pts.append(lower_left_pt)
+    return LineString(pts)
+
+# def make_grid(lower_left_pt, h=3, w=3, delta=1):
```

Comment:
remove these commented out blocks

Reply:
test_reblock still needs some work and I need to build out some additional unit tests so this is still wip

Reply:
i don't think WIP stuff should be committed to master - it should live on a separate feature branch